pkg/ddc/eac/utils.go

```diff
@@ -113,11 +114,19 @@ func (e *EACEngine) getWorkerPods() (pods []v1.Pod, err error) {
 		return pods, err
 	}
 
-	pods, err = kubeclient.GetPodsForStatefulSet(e.Client, sts, selector)
+	allpods, err := kubeclient.GetPodsForStatefulSet(e.Client, sts, selector)
 	if err != nil {
 		return pods, err
 	}
 
+	for _, pod := range allpods {
+		if !podutil.IsPodReady(&pod) {
+			e.Log.V(1).Info("Skip the pod because it's not ready", "pod", pod.Name)
+			continue
+		}
+		pods = append(pods, pod)
```

Comment:
How about using

```golang
if podutil.IsPodReady(&pod) {
            pods = append(pods, pod)
        } else {
            e.Log.V(1).Info("Skip the pod because it's not ready", "pod", pod.Name)
        }
```

Reply:
Done